src/rules/no-empty-source/README.md

```diff
@@ -0,0 +1,33 @@
+# no-empty-source
+
+Disallow empty source. Sources containing only spaces or/and empty lines is considered empty.
```

Comment:
Please move "Sources containing only spaces or/and empty lines is considered empty." to below the example below.